decentraland/redux/middlewares.js

```diff
@@ -0,0 +1,60 @@
+import { applyMiddleware } from "redux";
+import {
+  SET_ACCOUNT,
+  SET_ACCOUNT_CREATION_STATUS,
+  UPDATE_ACTION_FOR_ACCOUNT_CREATION_STATUS,
+  SET_ACCOUNT_CREATION_ACTIONS,
+  ADD_TO_MY_ASSETS_LIST,
+  REMOVE_MY_ASSET_FROM_LIST,
+  SET_MY_ASSETS_LIST,
+} from "./actions";
+import {
+  storeAccount,
+  storeAccountCreationStatus,
+  storeAccountCreationActions,
+  storeMyAssets,
+} from "@helpers/storage";
+
+const storer = store => next => async action => {
+  const { type } = action;
+  const result = next(action);
+  const nextState = store.getState();
```

Comment:
Did you decide that as long as it’s in middleware this isn’t a code smell?

Reply:
At first glance, the size doesn't seem too long for me, but we can split into smaller middlewares or to helpers functions if you prefer.

Reply:
No, it seems fine to me! I thought maybe you originally were concerned about the use of getStore for this purpose more so than any size-related concerns. Either way, feel free to resolve this. Thanks!

Reply:
Oh, got it now! :smile: 
Yes, since the middleware provides the store object I think that isn't a code smell.
Resolving this.

Reply:
Okay, great!